app/api/csv/typeParsers.js

```diff
@@ -1,9 +1,12 @@
 import entities from 'api/entities';
 import thesauris from 'api/thesauris';
-import { unique, emptyString } from 'api/utils/filters';
 import url from 'url';
 
+import { unique, emptyString } from 'api/utils/filters';
+import geolocation from './typeParsers/geolocation.js';
+
 export default {
+  geolocation,
```

Comment:
All other types are async functions, yet this one is not.  Will the "importEntity" line 13 that 'awaits' for this not fail here?
https://javascript.info/async-await

Reply:
Perhaps the missing implementation test on the typeParsers might solve the doubt?

Reply:
i think you can await any function as the tests demonstrates, but i will make it an async function for clarity